src/cmasher/utils.py

```diff
@@ -576,114 +594,58 @@ def create_cmap_overview(
         cmaps = list(cmrcm.cmap_d.values())
 
     # If sort is a string, obtain proper function
+    sort_key: Callable[[Colormap], SupportsOrdering] | None
     if isinstance(sort, str):
-        # Convert sort to lowercase
-        sort = sort.lower()
```

Comment:
this is a very minor breaking change; I couldn't find a way to preserve this *and* satisfy type checkers. I think it's more valuable to have a detailed `Literal` than `str` as a type hint so overall I think it's worth it.